packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseList.tsx

```diff
@@ -1,16 +1,16 @@
 import { spacingsPx, typography } from '@trezor/theme';
 import styled from 'styled-components';
 
-export const PassphraseList = styled.div`
+export const PassphraseList = styled.div<{ $gap?: 'small' | 'normal' }>`
```

Comment:
nit: I prefer extra type definitions. 

(reason even in this case: it tends to break the VSCode code-colorization, dont know whym but extracting type helps)